### PR TITLE
feat(trace-viewer): allow host and port to be specified

### PR DESCRIPTION
**Pull request by gobeli**
Currently the `--host` and `--port` cli options are only available for `show-report`. 
This change would allow the two options to be passed to the `show-trace` command.

### CODE REVIEW

**Comment by gobeli** [comment]
@microsoft-github-policy-service agree

File: packages/playwright-core/src/cli/cli.ts [hunk 1]
```diff
@@ -298,6 +298,8 @@ program
 program
     .command('show-trace [trace...]')
     .option('-b, --browser <browserType>', 'browser to use, one of cr, chromium, ff, firefox, wk, webkit', 'chromium')
+    .option('-h, --host <host>', 'Host to serve trace on', 'localhost')
+    .option('-p, --port <port>', 'Port to serve trace on', '9323')
```
Comment (pavelfeldman) [inline packages/playwright-core/src/cli/cli.ts:302]: This is a different value from what it used to be.

File: packages/playwright-core/src/server/trace/viewer/traceViewer.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ import { serverSideCallMetadata } from '../../instrumentation';
 import { createPlaywright } from '../../playwright';
 import { ProgressController } from '../../progress';
 
-export async function showTraceViewer(traceUrls: string[], browserName: string, headless = false, preferredPort?: number): Promise<BrowserContext | undefined> {
+export async function showTraceViewer(traceUrls: string[], browserName: string, headless = false, host?: string, preferredPort?: number): Promise<BrowserContext | undefined> {
```
Comment (pavelfeldman) [inline packages/playwright-core/src/server/trace/viewer/traceViewer.ts:29]: Let's turn this into the options: { headless?: boolean, host?: string, port?: number }

**Review by pavelfeldman — COMMENTED** [review]
Looks good, a couple of small comments.

File: packages/playwright-core/src/cli/cli.ts (hunk 1 above)
Comment (gobeli) [reply packages/playwright-core/src/cli/cli.ts]: At `9322` now

File: packages/playwright-core/src/server/trace/viewer/traceViewer.ts (hunk 2 above)
Comment (gobeli) [reply packages/playwright-core/src/server/trace/viewer/traceViewer.ts]: Options object is now used for `headless`, `host` and `port`

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**